ecl/hqlcpp/hqlhtcpp.cpp

```diff
@@ -10189,7 +10189,8 @@ void HqlCppTranslator::buildFormatCrcFunction(BuildCtx & ctx, const char * name,
 {
     IHqlExpression * payload = expr ? expr->queryAttribute(_payload_Atom) : NULL;
     // MORE - do we need to keep this consistent - if so will have to trim out the originals and the filepos
-    OwnedHqlExpr exprToCrc = getSerializedForm(dataset->queryRecord(), diskAtom);
+//    OwnedHqlExpr exprToCrc = getSerializedForm(dataset->queryRecord(), diskAtom);
```

Comment:
@ghalliday Is it worth just removing these lines now?  Can't see a reason why the older CRC would be needed and, if it is later found to be useful, it could be added back it.

Reply:
I will delete